channels/il.m3u

```diff
@@ -1,51 +1,37 @@
+https://kanlivep2event-i.akamaihd.net/hls/live/747600/747600/source1_4k/chunklist.m3u8
+#EXTINF:-1 tvg-id="Kan 23-IS" tvg-name="IL: Kan 23" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6b/KanHinuchit.svg/2880px-KanHinuchit.svg.png" group-title="Kids",ch 23 החינוכית  
+https://kanlivep2event-i.akamaihd.net/hls/live/747600/747600/source1_2.5k/chunklist.m3u8
+#EXTINF:-1 tvg-id="Music 24-IS" tvg-name="Music 24-IS" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/he/thumb/f/fa/Ch24telad.svg/1920px-Ch24telad.svg.png" group-title="Music",Music 24 ערוץ
+https://keshethlslive-lh.akamaihd.net/i/24live_1@195271/index_1400_av-b.m3u8
+#EXTINF:-1 tvg-id="Sport 5-IS" tvg-name="IL: Sport 5 HD" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/he/thumb/a/ad/Sport5ch.svg/1920px-Sport5ch.svg.png" group-title="Sport",Sport אתר ספורט 5
+https://sport5-lh.akamaihd.net/i/radiolivev_0@698733/index_1_av-p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/he/thumb/5/53/News12_2019new.svg/2880px-News12_2019new.svg.png" group-title="News",Channel 2 News חדשות
 http://keshethlslive-lh.akamaihd.net/i/c2n_1@195269/index_2200_av-p.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/NmnXn2z.jpg" group-title="News",Channel 2 News
-http://keshethlslive-lh.akamaihd.net/i/c2n_1@195269/master.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/NmnXn2z.jpg" group-title="News",Channel 2 News
-http://keshethlslive-lh.akamaihd.net/i/c2nsec_1@195268/master.m3u8
-#EXTINF:-1 tvg-id="Channel 20-IS" tvg-name="IL: ISRAEL Channel 20" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/Jsck6fw.jpg" group-title="News",Channel 20
+#EXTINF:-1 tvg-id="Channel 20-IS" tvg-name="IL: ISRAEL Channel 20" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Arutz_20.svg/1920px-Arutz_20.svg.png" group-title="Religious",Channel 20 ערוץ
 http://live.ch20-cdnwiz.com/ch20-abr/ch20-abr/playlist.m3u8
-#EXTINF:-1 tvg-id="Hidabroot Channel-IS" tvg-name="Hidabroot Channel-IS" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/wKi97Pg.png" group-title="",Hidabroot
+#EXTINF:-1 tvg-id="Hidabroot Channel-IS" tvg-name="Hidabroot Channel-IS" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/he/7/7a/Hidabroot_logo.png" group-title="Religious",Hidabroot
 http://stream5.shidur.net/htvlive2/_definst_/smil:live2.smil/chunklist.m3u8
-#EXTINF:-1 tvg-id="Hidabroot Channel-IS" tvg-name="Hidabroot Channel-IS" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/wKi97Pg.png" group-title="",Hidabroot
-https://stream71.shidur.net/htvlive2/_definst_/smil:live2.smil/chunklist_b1248000.m3u8
```

Comment:
This link works.

Reply:
I put Stream 5, this is stream71. Maybe there is also stream 6,7,8,9...999 - do you want to put all of them? I think it would be confusing from a user experience standpoint.. For me as a user I would prefer just one stream that works well.
This is your repository, so whatever you decide, I will comply

Reply:
We keep all broadcast options as a backup.

Reply:
Ok, I will add them, 
but I think the outcome is lower usability - it will be a cluttered list with a lot of duplicates and irrelevant stuff, instead of a curated list of just the things that people are looking for.